### DISCUSSION TITLE
반성합니다...

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#589] [6. 반성문]
요즘 사내 롤대회가 열려서 롤 연습하랴, 스크림하랴 정신이 없네요

이번주도 반성문을 써야 할 것 같네요...

대신 꼭 우승을 가져오겠습니다 ~ !

**Comment by shkisme** [comment]
ㄷㄷㄷㄷ 카카오 롤대회
